src/components/molecules/DepartmentManagement/DepartmentManagement.tsx

```diff
@@ -0,0 +1,60 @@
+import { FC, useCallback } from 'react'
```

Comment:
**[blocker]** There seem to be some other issues with changing :

1. When I try to add 1 new department and I leave it empty, I will get a success notification, the modal will close, but the new department with empty name is not added. If I'm not mistaken we should show error there instead and not allow submitting the form.

2. Similar issue, when I change the name of an existing department to an empty string. It will show a success notification, as if everything worked, however that department is actually just ignored and the value doesn't get changed + user won't see an error

Reply:
**[blocker]**

This issue still exists to some degree, it allows be to submit a form with new empty departments and changing existing departments to empty strings, however it does nothing other than closing the modal. This can be very confusing for a user (No idea if the changes went through or not + once they realise the changes didn't go through they still don't know why)

Reply:
I'm currently adding a "No Changes Made" notification. I think if we can find a solution to use react-hook-form useFieldArray to add and remove values and fields, then we can use submitt button disable solution